app/views/categories/show.html.erb

```diff
@@ -0,0 +1,69 @@
+<% render @category %>
+
+<nav class="header-categories d-flex flex-center">
+  <div class="icons">
+    <%= link_to  categories_path do %>
+      <i class="fa-solid fa-right-to-bracket"></i>
+    <% end %>
+  </div>
+  <h1>CATEGORY DETAILS</h1>
+</nav>
```

Comment:
- [ ] How about using the backward icon or other icons to show the back button.

![image](https://user-images.githubusercontent.com/55635977/154809706-aaa18ecd-070d-4481-9134-ddca41571665.png)

Reply:
Hi @Emmyn5600,
I have implemented all changes requested right
Thank you very much to contribute on my professionalism, 🙏🏻